R/label_roche.R

```diff
@@ -35,3 +35,49 @@ label_roche_pvalue <- function(big.mark = ifelse(decimal.mark == ",", " ", ","),
                                ...) {
   function(x) roche_pvalue(x, big.mark = big.mark, decimal.mark = decimal.mark, ...)
 }
+
+#' Round percentages
+#'
+#' @inheritParams gtsummary::style_percent
+#'
+#' @return A character vector of rounded percent values
+#' @name roche_percent
+#'
+#' @examples
+#' x <- c(0.0000001, 0.123456)
+#'
+#' roche_percent(x)
+#' label_roche_percent()(x)
+NULL
+
+#' @export
+#' @rdname roche_percent
+roche_percent <- function(x,
+                          big.mark = ifelse(decimal.mark == ",", " ", ","),
+                          decimal.mark = getOption("OutDec"),
+                          ...) {
+  percent <- x * 100
+
+  dplyr::case_when(
+    is.na(percent) ~ NA_character_,
+    abs(percent - 100) < 1e-8 ~ "100%",
```

Comment:
Can you remove the default `"%"` at the end so that it operates more similarly to the other stying functions in gtsummary. You can add the prefix and suffix arguments too. Thanks!